Bio/SeqFeature.py

```diff
@@ -292,7 +300,7 @@ def _flip(self, length):
                           qualifiers=OrderedDict(self.qualifiers.items()))
 
     def extract(self, parent_sequence):
-        """Extract feature sequence from the supplied parent sequence.
+        """Extract the sequence feature from the supplied parent sequence.
```

Comment:
Rather than ``sequence feature``, I would keep ``feature sequence`` or perhaps use ``feature's sequence``. The key point is this method returns a sequence (not a feature object).